kolena/workflow/test_sample.py

```diff
@@ -109,6 +109,7 @@ class _TestSampleType(DataType):
     VIDEO = "VIDEO"
     DOCUMENT = "DOCUMENT"
     COMPOSITE = "COMPOSITE"
+    POINTCLOUD = "POINTCLOUD"
```

Comment:
Requesting `POINT_CLOUD` to align with the [corresponding asset type](https://github.com/kolenaIO/kolena/blob/trunk/kolena/workflow/asset.py#L42)